spec/requests/user_pages_spec.rb

```diff
@@ -1,416 +1,416 @@
+        current_path.should == user_path(user)
+      end
+    end
+  end
+
+  describe "INDEX page" do
+    let!(:user) { create(:user, name: 'Andy', email: 'andy@example.com') }
+    let!(:admin) { create(:admin_user) }
+
+    describe "as an admin" do
+      before do
+        sign_in admin
+        visit users_path
+      end
+
+      it { should have_selector('h1', text: 'Manage Users') }
+      it { should have_selector('th', text: 'Name') }
+      it { should have_selector('th', text: 'Email') }
+
+      it "has user fields displayed in table", js:true do
+        should have_selector('td', text: 'Andy')
+        should have_selector('td', text: 'andy@example.com')
+        should have_selector('td', text: admin.name)
+        should have_selector('td', text: admin.email)
+      end
+
+      it "should link to new" do
+        click_link "New User"
+        current_path.should == new_user_path
+      end
+
+      it "should link to show", js: true do
+        within(:xpath, '//tr[contains(.//td/text(), "Andy")]') do
+          page.find(".fa-eye").click
+          current_path.should == user_path(user)
+        end
+      end
+
+      it "should link to edit", js: true do
+        within(:xpath, '//tr[contains(.//td/text(), "Andy")]') do
+          page.find(".fa-pencil").click
+          current_path.should == edit_user_path(user)
+        end
+      end
+
+      it "should link to destroy", js: true do
+        page.evaluate_script('window.confirm = function() { return true; }')
+        within(:xpath, '//tr[contains(.//td/text(), "Andy")]') do
+          page.find(".fa-trash").click
+          current_path.should == users_path
+        end
+        page.should have_selector('.flash-success', text: 'User "Andy" was successfully deleted')
+        page.should_not have_selector('td', text: 'Andy')
+        page.should_not have_selector('td', text: 'andy@example.com')
+        page.should_not have_selector('td.delete', text: 'false')
+      end
+    end
+
+    describe "as a non-admin" do
+      before do
+        sign_in user
+        visit users_path
+      end
+
+      it "is inaccessible" do
+        current_path.should == root_path
+        page.should have_selector('.flash-alert', text: 'You are not authorized to access this page.')
+      end
+    end
+  end
+
+  describe "FEEDBACKS" do
+    let(:reviewer) { create(:user) }
+    let(:ac) { create(:associate_consultant) }
+    let(:review) { create(:review, associate_consultant: ac) }
+    let!(:feedback) { create(:feedback, review: review, user: reviewer) }
+
+    describe "as the reviewer" do
+      before do
+        sign_in reviewer
+      end
+
+      it "displays the feedback with a 'Resume Feedback' action if not submitted" do
+        visit feedbacks_user_path(reviewer)
+        page.should have_selector('#feedback-requests td', text: ac.user.name)
+        page.should have_selector('#feedback-requests td', text: review.review_type)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.